### DISCUSSION TITLE
Function for leaving runtime in full screen mode.

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#1540] [Ideas]
Hi. Using the "Log full mode" setting in Enabled mode is very useful when we want to hide the navigation menu. If we also hide the header, the question is how do we leave runtime mode in this situation?
It would be preferable to have a function that can be called on the action of a button and that allows us to leave runtime mode.
<img width="467" alt="Screenshot 2024-12-14 at 13 49 19" src="https://github.com/user-attachments/assets/b64ed795-c9db-4d54-8915-5822722e0205" />

**Comment by unocelli** [comment]
Hi, If active, it also records info logs, which can be many and useless
